chain-impl-mockchain/src/config.rs

```diff
@@ -765,7 +765,9 @@ impl TagLen {
 #[cfg(any(test, feature = "property-test-api"))]
 mod test {
     use super::*;
-    use quickcheck::{Arbitrary, Gen, TestResult};
+    #[cfg(test)]
+    use quickcheck::TestResult;
+    use quickcheck::{Arbitrary, Gen};
```

Comment:
It would be better if we kept the API parts that are enabled with `property-test-api` feature to a separate module, which I tried to insist is called `testing` (BTW, the conventional name for unit test modules is `tests`, not `test`, but I'm probably being too OCD about this). But I can't see how this can even export any API since the module is private.

It might help to build with `--features property-test-api` and resolve any warnings about unused things also there.

Reply:
I am putting my faith in the chain-generator for better API on the long run. This is problematic because of how the chain-libs expose property-test-api in order to help the scenario based testing.